chain/client/src/test_utils/test_env_builder.rs

```diff
@@ -360,10 +365,15 @@ impl TestEnvBuilder {
 
     /// Internal impl to make sure runtimes are initialized.
     fn ensure_runtimes(self) -> Self {
+        let state_snapshot_enabled = self.state_snapshot_enabled;
         let ret = self.ensure_epoch_managers();
         if ret.runtimes.is_some() {
             ret
```

Comment:
nit: Maybe return ret and get rid of the else? It would flatten the code a little.

Reply:
Separate PR, but yeah, good place for minor improvements